### DISCUSSION TITLE
Aesthetic disruption in  humpback

### DISCUSSION THREAD

**Discussion by Balnik** [FakeFishGames/Barotrauma#9723] [Ideas and suggestions]
maybe need fix?
![Barotrauma_3i06R0FuMp](https://user-images.githubusercontent.com/98221461/185190564-6f6a0459-bf74-4a5d-ab45-c3c9be8cbfb9.jpg)
![234243](https://user-images.githubusercontent.com/98221461/185190578-13e8d836-11a0-43f0-a6b0-7b1b89689a54.png)
![Barotrauma_yeWBUuvNod](https://user-images.githubusercontent.com/98221461/185190591-9dda0bff-7ec1-453b-a71c-9c0f2a0d8a80.jpg)
![Barotrauma_Q9zUCOpsfP](https://user-images.githubusercontent.com/98221461/185190607-8aca1ae1-cb14-4886-938d-1d0474c5e8c4.jpg)

**Comment by gerakis** [comment]
Whats wrong with it exactly?
